dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/Schema.java

```diff
@@ -162,6 +162,16 @@
      */
     private boolean dataShareable;
 
+    /**
+     * Is data write sharing support for instances of this class.
```

Comment:
Would appreciate a short detail on what data read and write sharable is in contrast to just data sharing for the two new fields.